custom/sensors/CANEncoder.java

```diff
@@ -164,12 +164,9 @@ public void reset() {
 		}
 	}
 
-	public void resetViaOffset(double setpoint) {
-		this.offset += setpoint - getDistance();
-	}
-
-	public void resetViaOffset() {
```

Comment:
Maybe keep resetSensorValue to run setSensorValue(0)?

Reply:
that exists